gradio/components.py

```diff
@@ -1241,7 +1241,9 @@ def __init__(
             if isinstance(value, str):
                 value = [value]
         if not multiselect and max_choices is not None:
-            warnings.warn("The `max_choices` parameter is ignored when `multiselect` is False.")
+            warnings.warn(
```

Comment:
Why is black complaining about this now? 🤔 

Did you update versions?

Reply:
Hmm where is it complaining? The GitHub actions are running fine (except for that one flaky test which I need to fix)

Reply:
I'm wondering why it started modifying this line out of a sudden if it was being linted before.

Reply:
Oh interesting, this change is not part of this PR anymore, so the same change must have happened in another PR that got merged in.

Reply:
It turns out that I did actually upgrade black versions. Have downgraded down to `black==22.6.0` to match what we have on CI. No complaints from black as far as I can tell, so I think we're good for now.